### DISCUSSION TITLE
i3 desktop status on pre-configured garuda distro

### DISCUSSION THREAD

**Discussion by MomoIsHeree** [i3/i3#5792] [Q&A] [answered]
Hey there!

I've been using the Garuda Distro with pre-configured i3 for a while, and I've been wondering if anyone knows where the desktop status comes from and how to configure it. I couldn't find anything in i3-status and also don't have i3-blocks, so I'm kind of lost.

Any help is appreciated, thank you.

Heres how it currently looks
![2023-11-27_22-58](https://github.com/i3/i3/assets/101149088/5626c968-3ec4-4c5f-ab2a-37cc9198cb25)

**Comment by okraits** [comment, marked as answer]
I guess that is conky.

**Reply by MomoIsHeree** [reply]
That is indeed conky! I found it myself after a bit more research. Sorry for the unnecessary post, but thanks for the help!
